Dockerfile

```diff
@@ -20,6 +20,11 @@ WORKDIR /install
 
 COPY requirements.txt /requirements.txt
 
+# Instructing pip to fetch wheels from piwheels.org" on ARMv6 and ARMv7 machines
```

Comment:
"In the future, when changedetection supports Python 3.11 we can unpin some requirements"

so that means that 3.11 does not need this?

Reply:
The mentioned line will be needed. When applied, we can review requirements.txt to remove pinned packages due to past missing wheels that are now available today.